data/sql/updates/pending_db_world/agamand.sql

```diff
@@ -0,0 +1,66 @@
+-- 
+DELETE FROM `creature` WHERE `id1` IN (1530, 1534, 1658);
```

Comment:
GUID is pri key. Delete GUID.

Reply:
In this instance I want all of them removed and replaced. Is this not allowed in this case? I mean I can put all the guids between brackets but I am also lazy

Reply:
You need to remove GUID, as it is the only primary key in this table. Otherwise, there is a risk for duplicate entries.